dotcom-rendering/src/web/components/EmailSignup.stories.tsx

```diff
@@ -25,14 +26,20 @@ export default {
 
 const hidePrivacyMessage = (): boolean => boolean('hidePrivacyMessage', false);
 
-export const Default = () => (
+const Frame = ({ children }: { children: ReactNode }) => (
 	<ContainerLayout
 		title="EmailSignup"
 		showTopBorder={true}
 		sideBorders={true}
 		padContent={false}
 		centralBorder="partial"
 	>
+		{children}
```

Comment:
Could we try to avoid passing children like this and instead use props? I find it difficult to follow what is inside the children object so if we could be explicit I would find that really helpful.

Reply:
Thanks Charlotte this makes sense, try either way or keep it as children if that makes most sense

Reply:
👍  I'd not heard of the decorator feature before, but it sounds like exactly what I was trying to achieve! 

I think the wrapper is helpful as it makes the story show how the signup block will appear in the grid at different breakpoints - I'll set up a decorator to do this nicely.